rabbitmq-native/src/main/java/org/ballerinalang/messaging/rabbitmq/util/ConnectionUtils.java

```diff
@@ -132,7 +132,7 @@ private static SSLContext getSSLContext(BMap secureSocket) {
             char[] trustPassphrase = cryptoTrustStore.getStringValue(RabbitMQConstants.KEY_STORE_PASS).getValue()
                     .toCharArray();
             String trustFilePath = cryptoTrustStore.getStringValue(RabbitMQConstants.KEY_STORE_PATH).getValue();
-            String tlsVersion = secureSocket.getStringValue(RabbitMQConstants.RABBITMQ_CONNECTION_TLS_VERSION)
+            String tlsProtocol = secureSocket.getStringValue(RabbitMQConstants.RABBITMQ_CONNECTION_TLS_PROTOCOL)
```

Comment:
So, shall we update like this?
```suggestion
            String protocol = secureSocket.getStringValue(RabbitMQConstants.RABBITMQ_CONNECTION_SECURE_SOCKET_PROTOCOL)
```